generated/nidigital/nidigital/session.py

```diff
@@ -2186,14 +2700,14 @@ def get_time_set_edge_multiplier(self, time_set_name):
     def get_time_set_name(self, time_set_index):
         r'''get_time_set_name
 
-        TBD
+        You must provide a ViChar array to serve as a buffer for the value. You pass the number of bytes in the buffer as the **nameBufferSize**. If the current value of the property, including the terminating NULL byte, is larger than the size you indicate in the buffer size, the method copies (buffer size - 1) bytes into the buffer, places an ASCII NULL byte at the end of the buffer, and returns the buffer size you must pass to get the entire value. For example, if the value is "123456" and the buffer size is 4, the method places "123" into the buffer and returns 7. If you want to call this method just to get the required buffer size, you can pass 0 for **nameBufferSize** and VI_NULL for the name.
```

Comment:
Another one that should not be part of public API - #1557

Reply:
Docs removed